pallets/refungible/src/erc.rs

```diff
@@ -25,14 +25,14 @@ use pallet_evm_coder_substrate::call;
 use crate::{Config, RefungibleHandle};
 
 #[solidity_interface(
-	name = "UniqueRFT",
+	name = "UniqueRefungible",
 	is(via("CollectionHandle<T>", common_mut, Collection),)
 )]
 impl<T: Config> RefungibleHandle<T> where T::AccountId: From<[u8; 32]> {}
```

Comment:
```suggestion
impl<T: Config> RefungibleHandle<T> {}
```
Тут нет методов которые на это полагаются

Reply:
А, точно